### PR TITLE
Convert registry to k8s.gcr.io

### PR DESCRIPTION
**Pull request by thockin**
This PR was auto-generated.  Please apply human expertise to review for correctness.

Followup to https://github.com/kubernetes/kubernetes/pull/54174

xref https://github.com/kubernetes/release/issues/281

### CODE REVIEW

File: grafana/RELEASES.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 - Support Grafana 4.4.1.
 
 ## 4.0.2 (04-01-2017)
-- Formalized the image name for every arch to `gcr.io/google_containers/heapster-grafana-ARCH:VERSION`
+- Formalized the image name for every arch to `k8s.gcr.io/heapster-grafana-ARCH:VERSION`
```
Comment (x13n) [inline grafana/RELEASES.md]: Substituting this in old release notes doesn't seem to be the right thing to do.

File: influxdb/RELEASES.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
 - Updated to version v1.1.1; bumped Godeps and modified some code in heapster to use the latest schema
 
 ## v0.13.0 (4.1.2016)
-- Formalized the image name for every arch to `gcr.io/google_containers/influxdb-grafana-ARCH:VERSION`
+- Formalized the image name for every arch to `k8s.gcr.io/influxdb-grafana-ARCH:VERSION`
```
Comment (x13n) [inline influxdb/RELEASES.md]: ditto

**Comment by thockin** [comment]
Reverted those parts

**Pull request merged**